common/oshinko-get-cluster/oshinko-get-cluster.go

```diff
@@ -55,7 +54,7 @@ func createCluster(client *oclient.OshinkoRest, name string) (*models.ClusterMod
 		if err != nil || res == nil {
 			return nil, err
 		}
-		if int64(len(res.Pods)) != m + w {
+		if int64(len(res.Pods)) != res.Config.MasterCount + res.Config.WorkerCount {
```

Comment:
Note that we depend on the config object returned in the create response to include the configured count for the nodes, not the current count.  oshinko-get-cluster has no other way of knowing how many workers to wait for ...

This will have to be reconciled with other REST respsonses (like cluster lookup) that return a count based on current nodes.